cxx/envoy.mk

```diff
@@ -14,7 +14,7 @@ IS_PRIVATE ?= $(findstring private,$(_git_remote_urls))
   ENVOY_COMPILATION_MODE ?= opt
   # Increment BASE_ENVOY_RELVER on changes to `docker/base-envoy/Dockerfile`, or Envoy recipes.
   # You may reset BASE_ENVOY_RELVER when adjusting ENVOY_COMMIT.
-  BASE_ENVOY_RELVER ?= 2
+  BASE_ENVOY_RELVER ?= 1
```

Comment:
This seems odd?

Reply:
`478433e447e1bef3052c71b046161eb6bdab6abd` commit changed ENVOY_COMMIT from `a54d6eb82e8e83d65e52307e89557a07c0b6d1b0`  to `33ff697fc665b25463c7d64916ee6736782e2e76`. In the subsequent commit `478433e447e1bef3052c71b046161eb6bdab6abd` that merged master to this branch the `ENVOY_COMMIT` kept as previous commit but it just changed the `BASE_ENVOY_RELVER ?= 2` to `BASE_ENVOY_RELVER ?= 1`.   The Quay ambassador-base image [tagged](https://quay.io/repository/datawire/ambassador-base?tab=tags) as: `envoy-1.33ff697fc665b25463c7d64916ee6736782e2e76.opt`

Reply:
"You may reset BASE_ENVOY_RELVER when adjusting ENVOY_COMMIT."

Reply:
OK, thanks!